README.md

```diff
@@ -1,17 +1,17 @@
 <img src="logo-gopherblue.png" alt="" width="448" height="221">
 
-# The Go Cloud Project
+# The Go Cloud Development Kit (Go CDK)
 
 _Write once, run on any cloud ☁️_
 
 [![Build Status](https://travis-ci.com/google/go-cloud.svg?branch=master)][travis]
 [![godoc](https://godoc.org/github.com/google/go-cloud?status.svg)][godoc]
 [![Coverage Status](https://coveralls.io/repos/github/google/go-cloud/badge.svg?branch=master)](https://coveralls.io/github/google/go-cloud?branch=master)
 
-The Go Cloud Project is an initiative that will allow application developers to
-seamlessly deploy cloud applications on any combination of cloud providers. It
-does this by providing stable, idiomatic interfaces for common uses like storage
-and databases. Think `database/sql` for cloud products.
+The Go Cloud Development Kit will allow application developers to seamlessly
```

Comment:
Nit: I'd put (Go CDK) once again here after the full name to emphasize that this is the acronym we use, because we don't really introduce it in any other way (except the title parenthesized words)

Reply:
Done.